### ISSUE TITLE
``` Broadcast of Intent { flg=0x10008014 cmp=com.dragonscapes.global/com.unity.androidnotifications.UnityNotificationManager (has extras) } ```

### ISSUE THREAD

**Issue by qingfeng346** [Unity-Technologies/NotificationsSamples#68]
the anr error.
```
Broadcast of Intent { flg=0x10008014 cmp=com.dragonscapes.global/com.unity.androidnotifications.UnityNotificationManager (has extras) }
```

**Comment by aurimasc** [comment]
Looks like you also reported this to main repo, closing this one.

**Comment by qingfeng346** [comment]
what mean? is it fixed ? but my version is latest(1.4.2), it's still make anr!
